packages/appium/lib/utils.js

```diff
@@ -33,6 +34,18 @@ function inspectObject (args) {
   }
 }
 
+function parseExtensionArgs (extensionArgs, extensionName) {
+  if (!_.isString(extensionArgs)) {
+    return {};
+  }
+  const parsedExtensionArgs = parseJsonStringOrFile(extensionArgs);
+  const extensionSpecificArgs = parsedExtensionArgs[extensionName] ? parsedExtensionArgs[extensionName] : {};
```

Comment:
the ternary operator usage is redundant here. `const extensionSpecificArgs = parsedExtensionArgs[extensionName]` is enough

Reply:
otherwise the following condition looks suspicious